en/docs/Learn/Analytics/configuring-apim-analytics.md

```diff
@@ -112,125 +113,264 @@ Follow the instructions below to do the required configurations for WSO2 API-M t
-
+<html><div class="admonition info">
+<p class="admonition-title">Info</p>
+<p>It is required to change the values <code>receiver_username</code> and <code>receiver_password</code> parameters if the user of WSO2 API-M is different to the user of WSO2 API-M Analytics. 
+The default value <code>${admin.password}</code> retrieves the admin user's password of the current API-M node as the <code>receiver_password</code>.</p>
+</div>
+</html>
 </div></td>
 </tr>
 <tr class="even">
-<td><code>                  &lt;StreamProcessorRestApiURL&gt;                 </code></td>
-<td><code>                  https://&lt;host&gt;:&lt;port&gt;                 </code></td>
+<td><code>store_api_url</code></td>
+<td><code>https://&lt;host&gt;:&lt;port&gt;</code></td>
 <td>The WSO2 API-M Analytics REST API URL. The WSO2 API-M Analytics REST API connection information, which are included under the REST API-M connection information, are defined as global properties, as they are common to all the WSO2 API-M analytics.</td>
 </tr>
 <tr class="odd">
-<td><code>                  &lt;StreamProcessorRestApiUsername&gt;                 </code></td>
+<td><code>store_api_username </code></td>
 <td>A valid administrator username</td>
 <td>The administrator username to log into the remote WSO2 API-M Analytics server.</td>
 </tr>
 <tr class="even">
-<td><code>                  &lt;StreamProcessorRestApiPassword&gt;                 </code></td>
+<td><code>store_api_password</code></td>
 <td>The password of the username specified.</td>
 <td>The administrator password to log into the remote WSO2 API-M Analytics server.</td>
 </tr>
 </tbody>
 </table>
 
-4.  Save the changes.
+Save the changes.
 
 #### Step 4 - Configure databases
 
-Configuring databases allow you to persist data relating to APIs, process them and analyze. Follow the procedure below to configure databases. !!! info
-The following is a list of database versions that are compatible with WSO2 API-M Analytics.
-
--   Postgres 9.5 and later
--   MySQL 5.6
--   MySQL 5.7
--   Oracle 12c
--   MS SQL Server 2012
--   DB2
+Configuring databases allow you to persist data relating to APIs, process them and analyze. Follow the procedure below to configure databases. 
 
 1.  Stop the WSO2 API-M Analytics server if it is running already.
-2.  To configure the worker, open the `<API-M_Analytics_HOME>/conf/               dashboard               /deployment.yaml` file and edit the `APIM_ANALYTICS_DB` section. A sample for MySQL is shown below.
-
-    ``` java
-         - name: APIM_ANALYTICS_DB
-              description: "The datasource used for APIM statistics aggregated data."
+2.  Configure the dashboard profile.
+    1. Open the `<API-M_ANALYTICS_HOME>/conf/dashboard/deployment.yaml` file.
+    2. Edit the `WSO2_DASHBOARD_DB, APIM_ANALYTICS_DB` and `AM_DB` sections and point to your desired type of database. 
```

Comment:
Shouldn't we add the permission db as well?

Reply:
That is there under the 4 th point.